frontend/src/components/landing_page/LandingPage.js

```diff
@@ -112,50 +129,53 @@ function LandingPage() {
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                </div>
-                <div className="row">
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                    <div className="col">
-                        <img src={FeatureCollectionPhoto3}/>
-                        <h2>Shoes collection</h2>
-                        <h3>Start from <b className="textPurpleBold">price</b></h3>
-                    </div>
-                </div>
+                {featured.length > 0 ?
+                    <div className="row">
+                        <div className="col">
+                            <img src={FeatureCollectionPhoto3}/>
+                            <h2>{featured[1].name}</h2>
+                            <h3>Start from <b className="textPurpleBold">{featured[1].startingPrice}$</b></h3>
+                        </div>
+                        <div className="col">
+                            <img src={FeatureCollectionPhoto3}/>
+                            <h2>{featured[2].name}</h2>
+                            <h3>Start from <b className="textPurpleBold">{featured[2].startingPrice}$</b></h3>
+                        </div>
+                        <div className="col">
+                            <img src={FeatureCollectionPhoto3}/>
+                            <h2>{featured[3].name}</h2>
+                            <h3>Start from <b className="textPurpleBold">{featured[3].startingPrice}$</b></h3>
```

Comment:
All of these containers are almost identical.
What if we have 50+ featured products ?

Think abut extracting this